### PR TITLE
Add testbed for grafana

### PR DESCRIPTION
**Pull request by AlessandroCotto**
Testbed for grafana with following custom settings:
- custom admin credential
- disabled login ratelimit

### CODE REVIEW

File: weak_credentials/grafana/grafana.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Grafana
+
+# Note
+The `Dockerfile.Grafana` contains the following customisation:
```
Comment (dawidg-doyen) [inline weak_credentials/grafana/grafana.md]: typo, customization

File: weak_credentials/grafana/grafana.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- the admin user is created with the credential `admin:qwertyuiop` instead of the usual `admin:admin`
+- grafana is started with disabled brute force login protection (`GF_SECURITY_DISABLE_BRUTE_FORCE_LOGIN_PROTECTION=TRUE`) - by default it is enable
```
Comment (dawidg-doyen) [inline weak_credentials/grafana/grafana.md]: typo, "it is enabled"

File: weak_credentials/grafana/grafana.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+# Setup
+1. Ensure the files `runAndBuildGraphana.sh` and `Dockerfile.Grafana` are in the same folder
+
+2. Create docker images with this command: `chmod +x runAndBuildGraphana.sh && ./runAndBuildGraphana.sh`
+
+3. Connect to http://localhost:8873/ via browser
+
+4. Verify that grafana is running correctly by logging in with credential `admin:qwertyuiop`
```
Comment (dawidg-doyen) [inline weak_credentials/grafana/grafana.md]: credentials

File: weak_credentials/grafana/grafana.md (hunk 1 above)
Comment (AlessandroCotto) [reply weak_credentials/grafana/grafana.md]: Fixed

File: weak_credentials/grafana/grafana.md (hunk 2 above)
Comment (AlessandroCotto) [reply weak_credentials/grafana/grafana.md]: Fixed

File: weak_credentials/grafana/grafana.md (hunk 3 above)
Comment (AlessandroCotto) [reply weak_credentials/grafana/grafana.md]: Fixed

**Pull request merged**